catalog/ui/search-ui/standard/src/main/webapp/js/view/preferences/PreferencesModal.view.js

```diff
@@ -320,9 +320,10 @@ define([
             ui: {
                 tbody: 'tbody'
             },
+            viewComparator: 'label',
             initialize: function() {
                 this.collection.each(function(model) {
```

Comment:
You don't need to loop over the models here, you can listen to the collection instead.
```
this.listenTo(this, 'change:alpha', this.updateSort)
```
Or you could bake it in like this:
```
{
   viewComparator: 'label',
   collectionEvents: {
    "change:alpha": "updateSort"
  }
}